server/app/services/rpc/node_service_pod_handler.rb

```diff
@@ -0,0 +1,48 @@
+module Rpc
+  class NodeServicePodHandler
+    include Celluloid
+
+    def initialize(grid)
+      @grid = grid
+    end
+
+    # @param [String] id
+    # @return [Array<Hash>]
+    def list(id)
+      node = @grid.host_nodes.find_by(node_id: id)
+      return { error: 'Node not found' } unless node
+      return { error: 'Migration not done' } unless migration_done?
```

Comment:
Cases where migration fails -> no scheduled instances -> agents wipes every service instance... I just wanted to be sure that this cannot happen.